vtexdocs/devportal
# Feat/Lighthouse

#### What is the purpose of this pull request?

To update lighthouse action to run on multiple pages and add a pretty summary report.

#### What problem is this solving?

The Lighthouse action only runs in the home page.

#### How should this be manually tested?

Check out the output of the lighthouse action and make sure there is a comment in this PR with the summary of the run results.

#### Types of changes
